### ISSUE TITLE
update formula to be compliant with ```fluorine``` release

### ISSUE THREAD

**Issue by aboe76** [saltstack-formulas/apt-formula#51]
Warnings:
The following package(s) are "virtual package" names: apt-transport-https. These will no longer be supported as of the Fluorine release. Please update your SLS file(s) to use the actual package name.
```
apt-transport-https:
    pkg.installed
```
should be:
```
apt-transport-https:
    pkg.installed:
      - name: apt-transport-https
```

**Comment by javierbertoli** [comment]
`apt-transport-https` can be dropped in current distros, it's been merged into `apt` since version 1.5 (current Debian's stable has version 1.8.2). 

It should probably be kept around for ubuntu 16.04 or debian 8, so I think it should be moved to a `osfingermap.yaml` file.

**Comment by daks** [comment]
> Warnings:
> The following package(s) are "virtual package" names: apt-transport-https. These will no longer be supported as of the Fluorine release. Please update your SLS file(s) to use the actual package name.
> 
> ```
> apt-transport-https:
>     pkg.installed
> ```
> 
> should be:
> 
> ```
> apt-transport-https:
>     pkg.installed:
>       - name: apt-transport-https
> ```

I think there is no difference in both code, as Salt uses 'ID' as default 'name' if it's not explicitely available. I'll try to take some time to confirm this.

**Comment by aboe76** [comment]
@daks see : https://docs.saltstack.com/en/develop/topics/releases/2019.2.0.html#virtual-package-support-dropped-for-apt

**Comment by daks** [comment]
@aboe76 I saw the doc, but I don't understand it like you. It just states that virtual packages like `nginx` should not be used anymore with Salt on Debian/Ubuntu, but instead you should use the "real" package providing the virtual one.
I don't see anything talking about how pkg.installed handles the "name" or "ID" parameter.

Do you have some code to test it? I'm trying to reproduce it without success. _(I tried `nginx-formula` on 2019.2 and develop)_

**Comment by aboe76** [comment]
@dask, just use this formula on a Debian machine with 2019.2 installed,
my testbed is fresh debian 10 with salt-minion.

**Comment by daks** [comment]
I remember having seen this warning on our own Salt code, but now I can't find it in our CI logs. As we recently upgraded to 2019.2 it may have disappeared.

I tried a simple setup with a basic state
```
nginx:
  pkg.installed
```

with Debian 9 on 2018.3 and 2019.2 using the standard containers we use everywhere, and I can't reproduce it. See trace here https://friendpaste.com/5I9dfWtP56AEmhjgF3O5eK

I'm still searching...
